action.php

```diff
@@ -88,7 +88,7 @@ function _handle_newEntry(Doku_Event $event) {
         global $ID, $INFO;
 
         $ns    = cleanID($_REQUEST['ns']);
-        $title = str_replace(':', '', $_REQUEST['title']);
+        $title = str_replace([':', ';', '#', '&', '%', '/', '\\', '?'], '', $_REQUEST['title']);
```

Comment:
Done.

Note: I am not entirely happy with the direct access to `$_REQUEST` in this plugin. I think it might be better to user `class Input` or even `class Post` for this. OTOH it is unclear what _should_ happen if the `$_REQUEST` elements are missing or malformed? But that is a separate issue for later.